### ISSUE TITLE
Error loading 'C:\...\new page 1\page.html'.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fracte/keepnote#221]
```
What steps will reproduce the problem?
1. Create a NoteBook
2. Close it
3. Reopen it (this work)
4. click on a page to work on it

What is the expected output? What do you see instead?
The page should open
instead of that : "Error loading 'C:\Documents and Settings\xxxxxx\Mes
documents\docs KeepNote\test SANDBOX\new page 1\page.html'."

What version of the product are you using? On what operating system?
-> 0.6.2 on WinXP
-> reinstall of keepnote did not help
-> the data in the workbook are perfectly safe and accessible in tne pages.html
-> my workbook is structured with pages and sub-pages + attached files + links

Please provide any additional information below.


```

Original issue reported on code.google.com by `jeanmich...@gmail.com` on 25 Feb 2010 at 12:26

**Comment by GoogleCodeExporter** [comment]
```
Help / View error Logs gives : 

Could not open error log:
'utf8' codec can't decode bytes in position 25-27: invalid data

I'v not found any log in C:\Program Files 1\KeepNote\
```

Original comment by `jeanmich...@gmail.com` on 25 Feb 2010 at 12:32

**Comment by GoogleCodeExporter** [comment]
```
please look at comment at your first entry: 
http://code.google.com/p/keepnote/issues/detail?id=220

if this doesn't helped - may you attach your file here?
"C:\Documents and Settings\xxxxxx\Mes
documents\docs KeepNote\test SANDBOX\new page 1\page.html"
```

Original comment by `nudels0...@googlemail.com` on 27 Feb 2010 at 7:47

**Comment by GoogleCodeExporter** [comment]
```
Hello nudels : your comment helped, the issue is solved. The only point that 
remains,
is, that for 1 link flawed in 1 page, the entire notebook becomes no more 
accessible
and requires a repair through editing the flawed page, which may be difficult 
to finf. 

Would be nice if this behaviour could be improved ...
```

Original comment by `jeanmich...@gmail.com` on 1 Mar 2010 at 6:17

**Comment by GoogleCodeExporter** [comment]
Original comment by `matt.rasmus@gmail.com` on 15 Jun 2010 at 5:42
- Added labels: **encoding**

**Comment by GoogleCodeExporter** [comment]
```
Is this bug resolved in KeepNote 0.6.4?
```

Original comment by `matt.rasmus@gmail.com` on 10 Sep 2010 at 8:33

**Comment by GoogleCodeExporter** [comment]
Original comment by `matt.rasmus@gmail.com` on 14 May 2011 at 7:06
- Changed state: **Done**
